config/releases.exs

```diff
@@ -15,8 +15,6 @@ config :api_accounts,
   table_prefix: System.get_env("DYNAMO_TABLE_PREFIX"),
   migrate_on_start: true
 
-config :api_web, ApiWeb.Endpoint, secret_key_base: System.get_env("SECRET_KEY_BASE")
-
 config :api_web, :signing_salt, System.get_env("SIGNING_SALT")
```

Comment:
question: why not this secret as well?

Reply:
Heh, just to minimize the amount of AWS finagling I had to do. I had toyed with not moving any of the EB env vars, and just setting up the config provider for the future Keolis stuff, but figured I should have _something_ so I can test it end to end. But you're right, that I might as well move all the secrets in. Will updated.